### PR TITLE
[make verify] Display list of failed tests to the user at the end

### LINKED ISSUE #49845: [make verify] Display list of failed tests to the user at the end
**Is this a BUG REPORT or FEATURE REQUEST?**:
 /kind feature

**What happened**:
This prints a list of failed `make verify` tests, very usefull while investigating and fixing `make verify`.
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:
The linked [PR](#49720) is already being approved. This is a reference issue. 
**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration**:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by dhilipkumars**
**What this PR does / why we need it**:
Minor improvement to verify all script as it now displays list of failed tests at the end, makes it easier for fixing errors
```
$KUBE_VERIFY_GIT_BRANCH=someBranch hack/make-rules/verify.sh -v -Q
.
.
.
========================
FAILED TESTS
========================
hack/make-rules/../../hack/verify-boilerplate.sh
hack/make-rules/../../hack/verify-godep-licenses.sh
hack/make-rules/../../hack/verify-readonly-packages.sh
```

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #49845

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by dhilipkumars** [comment]
@shashidharatd , @madhusudancs Could one of you please have  look at this PR? this is pretty useful while investigating `make verify`  related errors.

File: hack/make-rules/verify.sh [hunk 1]
```diff
@@ -46,6 +46,7 @@ QUICK_PATTERNS+=(
   "verify-test-owners.sh"
 )
 
+
```
Comment (madhusudancs) [inline hack/make-rules/verify.sh]: Remove this.

File: hack/make-rules/verify.sh [hunk 2]
```diff
@@ -75,6 +76,19 @@ function run-cmd {
   fi
 }
 
+
```
Comment (madhusudancs) [inline hack/make-rules/verify.sh]: Remove this unnecessary blank line.

**Review by madhusudancs — COMMENTED** [review]
Mostly LGTM. A couple minor nits.

File: hack/make-rules/verify.sh (hunk 1 above)
Comment (dhilipkumars) [inline hack/make-rules/verify.sh]: done

File: hack/make-rules/verify.sh (hunk 2 above)
Comment (dhilipkumars) [inline hack/make-rules/verify.sh]: Done

**Comment by dhilipkumars** [comment]
/test pull-kubernetes-e2e-kops-aws
/test pull-kubernetes-e2e-gce-etcd3
--

**Comment by dhilipkumars** [comment]
@madhusudancs could you okay-to-test this PR please?

**Comment by dhilipkumars** [comment]
@shashidharatd all tests cleared could you approve, please?

**Comment by dhilipkumars** [comment]
@madhusudancs could you /approve no-issue please?

**Comment by dhilipkumars** [comment]
Flake : https://github.com/kubernetes/kubernetes/issues/49761

**Comment by cblecker** [comment]
.

**Comment by dhilipkumars** [comment]
@fejta, how to merge this PR it has been already approved, all the tests pass too

**Pull request merged**